torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -11,55 +12,57 @@
 from contextlib import contextmanager, suppress
 from datetime import timedelta
 from functools import reduce
-from typing import Union, NamedTuple, Callable, Any
+from typing import Any, Callable, NamedTuple, Union
```

Comment:
nit: Can we have formatting changes in a separate diff in the future? Makes it much easier to read. Also, this PR is good but in the future we might also want to split it up by something like 1) Add map_location arg 2) integrate with ZeRO to make each change self-contained.

Reply:
Absolutely. My bad, I will keep this in mind in the future.